sympy/functions/elementary/complexes.py

```diff
@@ -4,7 +4,7 @@
 from sympy.core.compatibility import u
 from sympy.core.exprtools import factor_terms
 from sympy.core.function import (Function, Derivative, ArgumentIndexError,
-    AppliedUndef)
+                                 AppliedUndef)
```

Comment:
Fixing indents for continuation lines is considered noise (PEP8 essentially allows any indent for them).
Please revert these.